ballerina/llm_client_azure_open_ai.bal

```diff
@@ -68,3 +132,7 @@ public isolated distinct client class AzureOpenAIModel {
         return parseResponseAsJson(resp);
     }
 }
+
+isolated function getJsonSchemaResponseTypeForAzureOpenAI(map<json> schema) returns AzureOpenAIResponseFormat|error {
+    return check getJsonSchemaResponseTypeForModel(schema).cloneWithType();
```

Comment:
Unnecessary check. Can also be an expression-bodied function. 

Please make sure Ballerina best practices are followed.